### ISSUE TITLE
FR: Let users configure which device is considered mobile (e.g. laptop with cellular)

### ISSUE THREAD

**Issue by Zocker1999NET** [tailscale/tailscale#4921]
### What are you trying to do?

I am using a laptop with cellular connection. There I discovered that tailscale seems to use a few kilobytes per second even when no application was sending/receiving any data (checked that no traffic was flowing through tailscale's interface). This resolves into (in my eyes) unnecessary data usage and so less data for me to use monthly.
On my phone, I never had the issue with tailscale using up much data by itself.
Because of this discovery and another comment I read (quoted below), I believe tailscale applies certain restrictions on mobile devices to save data & power (which is a great idea), however not on my laptop.

### How should we solve this?

Add a CLI parameter to `tailscale up` or option in the web UI to allow users like me to define their laptop as mobile device as well. This is especially good to have for any metered connections.

### What is the impact of not solving this?

In my case, I feat that tailscale is using up too much data and so that I need to disable tailscale when only connected to a cellular network which disconnects me from all my nodes and allows my cellular provider to see which web services I am using. If this does not get solved, I might need to migrate away from tailscale.

### Anything else?

> corp PR 2874 suppressed sending certain map poll updates to mobile clients
> Tailscale 1.16 disabled TCP keepalives
> 
> _Originally posted by @DentonGentry in https://github.com/tailscale/tailscale/issues/540#issuecomment-941979457_

From this comment, I understand that Tailscale behaves differently on devices considered mobile or not. That is why I propose this feature. So these improvements for mobile devices can be applied to laptops as well to save power & data usage.

Also, independent of this issue, I believe that solving #540 could improve my situation as well.

If I do not forget it, I will measure how much overhead traffic tailscale produces if my laptop is idle and only connected to cellular. I will post my results here later.

**Comment by Zocker1999NET** [comment]
In 5 minutes, while using my laptop to surf the net, I measured that 30.24 MiB were used over cellular (on wwan0 interface) while 27 MiB were flowing through tailscale's interface, meaning tailscale added an overhead of 3.24 MiB. That extrapolates to 38.88 MiB per hour and to over 1 GiB per month (= 1 hour / day for 30 days).
I do not know how much of that overhead was due to the WireGuard VPN itself (the wrapping of the packages) and how much was due to Tailscale (control & heartbeat traffic) itself, but IMO that is way too much (at least for German citizen). 1 GiB is a third of what I was allowed to use this month. This needs to be reduced massively.
